apps/snitch_core/priv/repo/migrations/20190206091901_update_general_config_seo_title.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Snitch.Repo.Migrations.UpdateGeneralConfigSeoTitle do
+  use Ecto.Migration
+
+  def change do
+    alter table("snitch_general_configurations") do
```

Comment:
What's the change here? Looks like the same thing is being added and removed.

Reply:
Earlier seo_title, frontend_url and backend_url didn't allow null values, but since these fields are optional I have modified the migration this way.

Reply:
adding default empty string

Reply:
so alter column should have done that, instead of removing the column. As removing the column would delete alla the values in DB.

Reply:
Yes, using [modify/3](https://hexdocs.pm/ecto_sql/Ecto.Migration.html#modify/3) to change the field type should be the correct approach. I missed it.